scene/3d/lightmapper.h

```diff
@@ -35,7 +35,7 @@
 
 #if !defined(__aligned)
 
-#if (defined(WIN32) || defined(_WIN32) || defined(__WIN32__) || defined(__NT__)) && !defined(__CYGWIN__)
+#if defined(_WIN32) && defined(_MSC_VER)
```

Comment:
Heh I'm pretty sure I had commented on the original PR that this was a very convoluted way to check `_MSC_VER`... looks like it was missed.